### PR TITLE
[OSASINFRA-2769] Add information that Kuryr doesn't support unidling

### PR DESCRIPTION
**Pull request by dulek**
Due to technical limitations of Octavia it is not possible to implement
automatic unidling with Kuryr when a Service gets called. This commit
adds note about this to the documentation.

### CODE REVIEW

**Comment by dulek** [comment]
@maxwelldb, please take a look.

File: modules/idle-unidling-applications.adoc [hunk 1]
```diff
@@ -25,3 +25,8 @@ $ oc scale --replicas=1 dc <dc_name>
 Automatic unidling by a router is currently only supported by the default
 HAProxy router.
 ====
+[NOTE]
+====
+Automatic unidling by a service is not supported when Kuryr-Kubernetes is
```
Comment (maxwelldb) [inline modules/idle-unidling-applications.adoc:30]: "unidling" is definitely the correct jargon to use here? Would love if it weren't, but if it is, it is what it is. :) @dulek

Comment (dulek) [reply modules/idle-unidling-applications.adoc:30]: It seems so, judging from the rest of the module I'm modifying. The note I'm adding is analogous to the one above.

File: modules/idle-unidling-applications.adoc [hunk 2]
```diff
@@ -25,3 +25,8 @@ $ oc scale --replicas=1 dc <dc_name>
...
+configured as SDN.
```
Comment (maxwelldb) [inline modules/idle-unidling-applications.adoc:31]:
```suggestion
If Kuryr-Kubernetes is configured as an SDN, automatic unidling by services is not supported.
```

Does this work?

Comment (dulek) [reply modules/idle-unidling-applications.adoc:31]: Sure!

**Comment by dulek** [comment]
@maxwelldb: The patch is updated with your suggestion.

**Comment by maxwelldb** [comment]
@dulek Thanks. Two more things:

1. Is there a BZ or Jira ticket that this change could be linked to?
2. Any suggestions for who would be best to QE this?

**Comment by dulek** [comment]
@maxwelldb:

1. We're tracking it as part of https://issues.redhat.com/browse/OSASINFRA-2769.
2. That would be @eurijon.
3. Any supported version really, so 4.6, 4.7, 4.8, 4.9, 4.10. I'd skip 3.11 as it goes EOL soon.

**Comment by eurijon** [comment]
It looks good to me :]

File: modules/idle-unidling-applications.adoc [hunk 3]
```diff
@@ -25,3 +25,7 @@ $ oc scale --replicas=1 dc <dc_name>
 Automatic unidling by a router is currently only supported by the default
 HAProxy router.
 ====
+[NOTE]
+====
+If Kuryr-Kubernetes is configured as an SDN, automatic unidling by services is not supported.
```
Comment (jldohmann) [inline modules/idle-unidling-applications.adoc:30]: Small change for active voice per ISG
```suggestion
Services do not support automatic unidling if you configure Kuryr-Kubernetes as an SDN.
```

**Review by jldohmann — COMMENTED** [review]
LGTM! Left a minor suggestion.

**Comment by maxwelldb** [comment]
@dulek I can merge this if you squash down to one commit.

**Comment by dulek** [comment]
@maxwelldb: Done!

**Comment by maxwelldb** [comment]
Done. @dulek

**Pull request merged**